languages/csharp/exercises/concept/nullability/.docs/instructions.md

```diff
@@ -0,0 +1,30 @@
+In this exercise you'll be writing code to helping printing name
+badges for factory employees.
+
+Employees have an id, a name and a department name. Badge labels are
+formatted as follows: `"[id] - [name] - [DEPARTMENT]"`
+
+### 1. Generate the badge label from employee properties
+
+Writing a static method called `Badge.Label()` to compute the string
+to be printed on the badge:
+
+```csharp
+Badge.Label(734, "Ernest Johnny Payne", "Strategic Communication");
+// => "[734] - Ernest Johnny Payne - STRATEGIC COMMUNICATION"
+```
+
+The department name is optional. If someone has no department, just
```

Comment:
Could you make this into a separate task? As we have task-specific hints, we can then point to particular resources for a task, in this case the `??` operator.